ruby/ruby-build/Portfile

```diff
@@ -3,24 +3,23 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        rbenv ruby-build 20220713 v
+github.setup        rbenv ruby-build 20221225 v
+revision            1
```

Comment:
Since you are bumping the version, you don't need to bump the revision.

Reply:
I had bumped the revision here because this is a *change* to the way that this port works. I‘m not sure what sort of signal that would require otherwise. I can change it back.

Reply:
Thanks for clarifying, but that's not what `revision` means in MacPorts. Bumping `revision` is for when `version` is *un*changed but something else about that port has changed (dependencies, installed files, default variants, etc.) such that the port needs to be rebuilt. It is expected to reset `revision` to 0 when bumping the `version`.

Reply:
Now that #17090 has been merged and this PR will no longer bump the `version`, you _should_ bump the `revision` due to the change in dependencies.